### ISSUE TITLE
[Bug] Stalagnate plants can generate without support on top or bottom

### ISSUE THREAD

**Issue by paulevsGitch** [quiqueck/BetterNether#9]
### What happened?

Stalagnates can generate without support, as a result there will be forest like this:
![2022-06-16_19 26 21](https://user-images.githubusercontent.com/17707380/174119966-0cf9d630-cdb6-4ffd-9148-cb25bad72a46.png)
![2022-06-16_19 34 48](https://user-images.githubusercontent.com/17707380/174121252-a26c4f84-f76a-4812-9623-df02362b633a.png)

### BetterNether

7.0.2

### BCLib

2.0.5

### Fabric API

0.56.0

### Fabric Loader

0.14.8

### Minecraft

1.19

### Relevant log output

```shell
No errors in log
```

### Other Mods

```shell
better-end-2.0.2
iris-mc1.19-1.2.5
sodium-fabric-mc1.19-0.4.2+build.16
```

**Comment by paulevsGitch** [comment]
Additional screen - they can also generate up to lava
![2022-06-16_19 38 09](https://user-images.githubusercontent.com/17707380/174121828-5d88e315-392d-4c60-8312-fc65aa8b8e58.png)

**Comment by quiqueck** [comment]
Should be fixed in the just release 7.0.3 update.
